src/config.ts

```diff
@@ -50,3 +50,7 @@ export enum EssServices {
 export function getEssServiceURI(service: EssServices) {
   return `https://${service}.ess.solid.integration.account.gov.uk`;
 }
+
+export function getCheckStoragePath(): string {
+  return "getCheckStoragePath()";
```

Comment:
Yup exactly, but this will need to be:
```suggestion
  return "private/govuk/identity/poc/credentials/vcs/";
```

Reply:
Also, I have a minor preference for not including the trailing slash here and instead putting it in the string interpolation because I think `${getCheckStoragePath()}/kbv/metadata` is slightly more legible. 

That's up to you though - this way does save us a character each time!

Reply:
oh bugger that's a find and replace, sorry should have spotted that.
Good point on the trailing slash, can fix that up.